code/modules/surgery/surgery.dm

```diff
@@ -169,6 +169,12 @@
 	var/can_infect = FALSE
 	/// How much blood this step can get on surgeon. See SURGERY_BLOODSPREAD_* defines
 	var/blood_level = SURGERY_BLOODSPREAD_NONE
+	//Sound played when the step is started
```

Comment:
Make sure to auto-doc these, and you should specify that they can be a single value or a list.

Reply:
Should be good now I think

Reply:
Yup, looks good